winter_openapi/generator.py

```diff
@@ -113,6 +120,36 @@ def __str__(self):
         return f'{method_path}: -> {self._return_type}: {self._message}'
 
 
+def get_url_path_without_prefix(url_path: str, path_prefix: Optional[str]) -> str:
+    # TODO: use removeprefix when python 3.9 will be used
+    path_prefix_stripped = path_prefix.strip('/')
+
+    if url_path.startswith(path_prefix_stripped):
+        url_path = url_path[len(path_prefix_stripped):]
+
+    return url_path
+
+
+def get_url_path_tag(route: Route, path_prefix: str) -> str:
+    path_prefix_segments = path_prefix.strip('/').split('/')
+    url_path_segments = route.url_path.strip('/').split('/')
+    path_prefix_segments = [segment for segment in path_prefix_segments if segment]  # remove empty segments like ['']
+
+    # for the cases with single route
+    if len(url_path_segments) <= len(path_prefix_segments):
+        if len(url_path_segments) == len(path_prefix_segments):
+            return ''
+        else:
+            raise ValueError(f'Invalid path prefix {path_prefix} for url_path {route.url_path}')
+
+    url_path_tag = url_path_segments[len(path_prefix_segments)]
+
+    if url_path_tag.startswith('{'):
+        return ''
```

Comment:
What is the branch for?

Reply:
For url: `prefix/{id}/get-resource` and prefix `/prefix`